MapEditorReborn/Commands/ToolgunCommands/CreateObject.cs

```diff
@@ -113,7 +113,12 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
 
                 if (data is not null)
                 {
-                    SpawnedObjects.Add(ObjectSpawner.SpawnSchematic(objectName, position, Quaternion.identity, Vector3.one, data));
+                    SchematicObject schematicObject;
+
+                    SpawnedObjects.Add(schematicObject = ObjectSpawner.SpawnSchematic(objectName, position, Quaternion.identity, Vector3.one, data));
+
+                    if (MapEditorReborn.Singleton.Config.AutoSelect)
+                        TrySelectObject(player, schematicObject);
```

Comment:
Why can't you call SelecteObject? Do you really need to create another method?

Reply:
If i do `ToolGunHandler::SelectObject`, the event SelectingObjectEventArgs does not fire that is why i need to create another method to do that/